.taskcluster.yml

```diff
@@ -134,7 +134,10 @@ tasks:
                 description: Schedules the build and test tasks for Application Services.
         "tasks_for == 'github-release' && event['action'] == 'published'":
           $let:
-            is_staging: 'event.repository.html_url != "https://github.com/mozilla/application-services"'
```

Comment:
This was setting `is_staging` to the string value `"event.repository.html != ..."`, rather than evaluating the condition